internal/mesh/internal/controllers/routes/controller.go

```diff
@@ -20,8 +21,30 @@ import (
 	"github.com/hashicorp/consul/proto-public/pbresource"
 )
 
+const (
```

Comment:
The routes controller was previously using the failover mapper as well. I also had to retrofit this controller to not use that any longer. So now this controller is partially cached indexer enabled but just for FailoverPolicy resources.